test/starlark_tests/verifier_scripts/archive_contents_test.sh

```diff
@@ -139,7 +139,10 @@ if [[ -n "${BINARY_TEST_FILE-}" ]]; then
 
   if [[ -n "${CODESIGN_INFO_CONTAINS-}" || -n "${CODESIGN_INFO_NOT_CONTAINS-}" ]]; then
     codesign_output="$(mktemp "${TMPDIR:-/tmp}/codesign_output.XXXXXX")"
-    codesign --display --verbose=4 "$path" &> "$codesign_output"
+    if ! codesign --display --verbose=4 "$path" &> "$codesign_output"; then
+      fail "Expected codesign --display to pass$newline$(cat $codesign_output)"
+    fi
+    cat "$codesign_output"
```

Comment:
Is this intentional? It'd be printed out twice if the test fails.

Reply:
unintentional